### ISSUE TITLE
Review annotations to cyclic nucleotide biosynthetic process

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-annotation#1979]
https://docs.google.com/spreadsheets/d/1YNy43WbMBRvPQvVX4AHmP-UkMYG5BqXRAqaENKbXJUU/edit#gid=0

**Comment by pfey03** [comment]
Removed the one we had, already had a process to signaling

**Comment by pgarmiri** [comment]
Hi, all the UniProt annotations were created at SIB. I can't alter them.  @sylvainpoux , could you please take care of those?
Thanks

**Comment by sylvainpoux** [comment]
I looked at the annotations (most of them were annotated by me) and I don't see the issue. CGAS is catalyzing cyclic GMP-AMP, a cyclic nucleotide essential for the defense response to virus. 
What is the reason behind modifying/deleting these annotations?
Sylvain

**Comment by pgarmiri** [comment]
Hi @sylvainpoux, my understanding is that the current terms imply involvement in metabolism but from the case you describe, it is a case on signalling in response to virus. Since CGAS is increasing the levels of cyclic GMP-AMP, I would probable annotate them to  GO:0043950 'positive regulation of cAMP-mediated signaling' and  GO:0010753 'positive regulation of cGMP-mediated signaling'.
Would that make sense?
 @pgaudet, is that right?

**Comment by pgarmiri** [comment]
Hi Sylvain,
Glad I could help!
Penelope

**Comment by deustp01** [comment]
Is it possible that two distinct things are being put together here? Cyclic nucleotide is synthesized from nucleotide triphosphate - that's a metabolic function or process. Now that it exists, the cyclic nucleotide modulates some other process, here something to do with a viral life cycle inside an infected host cell - that's a regulatory function or process. CGAS is an enzyme that catalyzes the metabolic reaction, so it should be annotated with  GO:0061501 cyclic-GMP-AMP synthase activity, whatever other annotations may or may not be appropriate. I would get (but definitely as an amateur in the annotation-of-regulation business), that CGAS should not be annotated as a regulator of virus life cycle, unless there are terms for "regulation as a downstream effect", because it's the cyclic nucleotide that is doing the regulating, not the CGAS. @ukemi ?

**Comment by pgaudet** [comment]
@deustp01 Could this be a case of 'acts upstream of regulation of virus life cycle'? 
@vanaukenk @ukemi @thomaspd

**Comment by deustp01** [comment]
That's true, but unless the virus is stimulating this generation of cyclic nucleotide it may be too far upstream. The virus uses all sorts of normal host cell metabolic products for its on purposes, and you may be on the way here to inferring that host ribosomal proteins are acting upstream, for example.

**Comment by pgaudet** [comment]
OK sorry I had missed your NOT in "CGAS should **not** be annotated as a regulator of virus life cycle'

I completely agree....
